examples/todos/src/main.rs

```diff
@@ -88,6 +89,9 @@ impl Application for Todos {
                     Message::InputChanged(value) => {
                         state.input_value = value;
                     }
+                    Message::InputPasted(value) => {
+                        state.input_value = value;
+                    }
```

Comment:
I don't think we should change the example. The behavior will be exactly the same as before, so we are just showcasing how to overcomplicate things.

Reply:
Ok. Will remove when I get home.

Reply:
Alright the test has been removed